packages/fast-components/src/styles/recipes.ts

```diff
@@ -0,0 +1,339 @@
+import {
+    accentFill,
+    accentFillLarge,
+    accentForeground,
+    accentForegroundCut,
+    accentForegroundLarge,
+    DesignSystem,
+    neutralDividerRest,
+    neutralFill,
+    neutralFillCard,
+    neutralFillInput,
+    neutralFillStealth,
+    neutralFillToggle,
+    neutralFocus,
+    neutralFocusInnerAccent,
+    neutralForeground,
+    neutralForegroundHint,
+    neutralForegroundHintLarge,
+    neutralForegroundToggle,
+    neutralForegroundToggleLarge,
+    neutralLayerCard,
+    neutralLayerCardContainer,
+    neutralLayerFloating,
+    neutralLayerL1,
+    neutralLayerL1Alt,
+    neutralLayerL2,
+    neutralLayerL3,
+    neutralLayerL4,
+    neutralOutline,
+} from "@microsoft/fast-components-styles-msft";
+
+export interface DesignSystemResolverEntry {
+    name: string;
+    resolver: (designSystem: DesignSystem) => string | number;
+}
+export const neutralforegroundrest: DesignSystemResolverEntry = {
```

Comment:
Could these be camel case? It's pretty hard to read where they are referenced.

Reply:
oops - yes.